include/error.h

```diff
@@ -51,4 +51,7 @@
 #define FAILURE		-1
 #endif
 
+#define RET_OK(x)	((x) == SUCCESS)
+#define RET_FAIL(x)	((x) != SUCCESS)
+
```

Comment:
Usually, the return code is checked if less than 0. In our case, we got only 2 return codes so we can use '!=' but maybe we should add our own return codes later on.